frontend/packages/topology/src/behavior/useDragNode.tsx

```diff
@@ -116,14 +116,16 @@ export const useDragNode = <
             }
           }
 
-          elementRef.current
-            .getController()
-            .fireEvent(
-              DRAG_NODE_END_EVENT,
-              elementRef.current,
-              monitor.getDragEvent(),
-              monitor.getOperation(),
-            );
+          try {
+            elementRef.current
+              .getController()
+              .fireEvent(
+                DRAG_NODE_END_EVENT,
+                elementRef.current,
+                monitor.getDragEvent(),
+                monitor.getOperation(),
+              );
+          } catch {} // eslint-disable-line no-empty
```

Comment:
We shouldn't be suppressing errors like this.

Reply:
ditto

Reply:
Why would we throw an error on drag end that we need to swallow it?

Reply:
Move call inside of a mobx `action` to prevent the exception from occurring.